README.md

```diff
@@ -24,6 +24,8 @@ CLUSTER_COUNT=<cluster count, optional, default to num of cpus>
 GAS_PRICE_CACHE_SECONDS=<seconds, optional, default to 0, and 0 means no cache>
 EXTRA_ESTIMATE_GAS=<eth_estimateGas will add this number to result, optional, default to 0>
 ENABLE_CACHE_ETH_CALL=<optional, enable eth_call cache, default to false>
+ENABLE_CACHE_ESTIMATE_GAS=<optional, enable eth_estimateGas cache, default to false>
```

Comment:
Why not enbale it by default?

Reply:
this pr follows the previous `eth_call` cache, since the cache is a bit complicated, we have introduced a on/off control in the original design. We can make it default true after running online for a while